.github/workflows/cronJob.yaml

```diff
@@ -0,0 +1,11 @@
+name: Cron Job
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
```

Comment:
I had to search around to learn that this means run every midnight. I wonder if we could add a comment which states that or whatever the current schedule is. Might even be good to have a link back to the docs: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onschedule so that someone who wants to change the schedule in the future can better understand the syntax.

Reply:
> I had to search around to learn that this means run every midnight. I wonder if we could add a comment which states that or whatever the current schedule is. Might even be good to have a link back to the docs: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onschedule so that someone who wants to change the schedule in the future can better understand the syntax.

Sure. Added the comment and the reference link